### ISSUE TITLE
does opatch automatically stop and start DB ?

### ISSUE THREAD

**Issue by winningpanthers** [biemond/biemond-oradb#184]
Looks like opatch is not stopping the DB before patching and then start it back up after patching. 
Is this working as expected? Any thoughts on how do I introduce stop , then patch and then start? I would hate to use exec . thanks

**Comment by biemond** [comment]
indeed, no stop unless opatch auto is doing this, because I'm doing it on a fresh install. patching on a running db will be hard, you need to do this at a certain moment, but you can use dbactions or db_control to do this yourself.

**Comment by winningpanthers** [comment]
thanks. yes there is a running db. How do I automatically stop DB for patching and then start again post patching? I tried [STOP]~> OPATCH -> [START], but i see stop and start not idempotent. any thoughts?

**Comment by biemond** [comment]
I guess a different manifest for patching, after that return to the original/updated one with the new patch.  I can add it to opatch but this can only go wrong , beter protect the apps who needs an active db and make this a separate controlled action.
